openverse_catalog/docs/data_models.md

```diff
@@ -0,0 +1,57 @@
+TODO: This documentation is temporary and should be replaced by more thorough documentation of our DB fields in https://github.com/WordPress/openverse-catalog/issues/783>
+
+# Data Models
+
+The following is temporary, limited documentation of the columns for each of our Catalog data models.
+
+## Required Fields
+
+| field name | description |
+| --- | --- |
+| *foreign_landing_url* | URL of page where the record lives on the source website. |
+| *audio_url* / *image_url* | Direct link to the media file. Note that the field name differs depending on media type. |
+| *license_info* | LicenseInfo object that has (1) the URL of the license for the record, (2) string representation of the license, (3) version of the license, (4) raw license URL that was by provider, if different from canonical URL |
+
+## Optional Fields
+
+The following fields are optional, but it is highly encouraged to populate as much data as possible:
+
+| field name | description |
+| --- | --- |
+| *foreign_identifier* | Unique identifier for the record on the source site. |
+| *thumbnail_url* | Direct link to a thumbnail-sized version of the record. |
+| *filesize* | Size of the main file in bytes. |
+| *filetype* | The filetype of the main file, eg. 'mp3', 'jpg', etc. |
+| *creator* | The creator of the image. |
+| *creator_url* | The user page, or home page of the creator. |
+| *title* | Title of the record. |
+| *meta_data* | Dictionary of metadata about the record. Currently, a key we prefer to have is `description`. |
+| *raw_tags* | List of tags associated with the record. |
+| *watermarked* | Boolean, true if the record has a watermark. |
+
+#### Image-specific fields
+
+Image also has the following fields:
+
+| field_name | description |
+| --- | --- |
+| *width* | Image width in pixels. |
+| *height* | Image height in pixels. |
+
+#### Audio-specific fields
+
+Audio has the following fields:
+
+| field_name | description |
+| --- | --- |
+| *duration* | Audio duration in milliseconds. |
+| *bit_rate* | Audio bit rate as int. |
+| *sample_rate* | Audio sample rate as int. |
+| *category* | Category such as 'music', 'sound', 'audio_book', or 'podcast'. |
```

Comment:
I believe this is available to all media types. Can we add a reference to it in the image-specific fields as well? Also it might be worth referencing the category enums we have: https://github.com/WordPress/openverse-catalog/blob/8423590fd86a0a3272ca91bc11f2f37979048181/openverse_catalog/dags/common/loader/provider_details.py#L118

(or, it looks like just enum? 🤔 doesn't seem like we have an audio category enum, I'll make an issue for that)

Reply:
Definitely! This documentation at the moment is basically just copying docstrings from the image stores. This doc in particular should be totally rewritten in the Data normalization project.

Reply:
I've left a lot of comments before I got to this one, @stacimc  :) Totally fine if we leave everything as is before the re-write. Except for the `foreign_identifier` being required.